VideoLocker/res/values/dimens.xml

```diff
@@ -59,7 +59,13 @@
     <dimen name="signup_social_title_margin_large">8dp</dimen>
     <dimen name="signup_social_title_margin_bottom">1dp</dimen>
 
+    <dimen name="edx_hairline">1px</dimen>
```

Comment:
Yes, we want to use the smallest size possible, that's what is meant by "hairline".